packages/@okta/vuepress-site/docs/guides/submit-app/faqs/index.md

```diff
@@ -2,138 +2,42 @@
 
-**What is Secure Web Authentication (SWA)?**
+**Q: What is Secure Web Authentication (SWA)? Is it different than using SAML?**
 
 SWA was developed by Okta to provide single sign-on for apps that do not support federated sign-on methods. Users can still sign in directly through the application and then enter their credentials for these apps on their Okta homepage. These credentials are stored so that users can access their apps with a single sign-on. When users first sign on to a SWA app from their homepage, they see a pop-up message asking if they were able to sign on successfully.
 
-**What is the process after I have submitted my app using the OIN Manager?**
-
-After you submit your app integration, it goes through a review by the Okta OIN Operations team, which reviews and tests your integration. We will reach out to you directly if your integration does not pass the review, if we need you to resubmit, or if we need additional information. Following the completed review, the integration is promoted to the OIN. Typical review time is two weeks, and you can track the progress of your submission in the [OIN Manager](https://oinmanager.okta.com).
-
-Have you submitted an app but have not seen a change in review status in the OIN Manager? Email <developers@okta.com>.
-
-**I'm an ISV setting up a SAML 2.0 app using the App Wizard and we have different domains for each customer. How do you manage these types of situations?**
-
-Currently, the App Wizard does not support custom domains, but public-facing integrations can still have this functionality. Create an app integration as you normally would for a single domain using the App Wizard. After you get to the step of submitting your app integration through the [OIN Manager](https://oinmanager.okta.com), you will be asked to fill out a form with all the relevant information.
-
-**My app currently supports WS-FED for single sign-on. Can I use the App Wizard?**
-
-The Okta App Wizard only supports SAML 2.0 for federated single sign-on. If your app supports WS-Fed, create a [WS-Fed Template App](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Configuring_WS_Federation). Afterwards, the template application you created can be used only within your account. In order to promote your template app to the Okta Integration Network, please email a screenshot with the details of the configured app to <developers@okta.com>, with your app name in the subject line.
-
-**I am creating a SWA app using the App Wizard but my application has fields on the login page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the login be configured using the App Wizard?**
+**Q: I am creating a SWA app using the App Wizard but my application has fields on the login page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the login be configured using the App Wizard?**
```

Comment:
I am creating a SWA app using the App Wizard but my application has fields on the **sign-in** page in addition to the standard username and password (for example: CustomerID, OrgID). Can an app with additional fields on the **sign-in page** be configured using the App Wizard?